Newark/index.js

```diff
@@ -1,66 +1,46 @@
-const swiperElemAnim = document.querySelector(".animation");
-const largeWoman = document.querySelector(".large-woman");
-const womanGallery = document.querySelector(".woman-gallery");
-const video = document.querySelector(".video");
-const screenshot = document.querySelector(".screenshot");
-const soundOn = document.querySelector(".soundOn");
-const soundOff = document.querySelector(".soundOff");
+// TurnON sound on the video
+const soundOn = document.querySelector(".sound-on");
+const soundOff = document.querySelector(".sound-off");
 
-// Устанавливаем параметры анимации
-swiperElemAnim.style.animation = "slide 8.5s ease-in-out forwards";
-swiperElemAnim.style.animationDelay = "1.5s";
-
-//ВКЛ ВЫКЛ звук на видео
 soundOff.addEventListener("click", () => {
   soundOff.style.display = "none";
   soundOn.style.display = "block";
```

Comment:
move it to styles

Reply:
muteButton.addEventListener("click", () => {
  if (video.muted) {
    soundOff.classList.add("hidden");
    soundOn.classList.remove("hidden");
  }
  else{
    soundOn.classList.add("hidden");
    soundOff.classList.remove("hidden");
  }
  video.muted = !video.muted;
}